src/fsipy/automatedPreprocessing/automated_preprocessing.py

```diff
@@ -430,7 +430,7 @@ def run_pre_processing(input_model, verbose_print, smoothing_method, smoothing_f
                                                    solid_thickness,
                                                    solid_thickness_parameters)
         except Exception:
-            print("ERROR: Mesh generation failed. Try to remesh with alternative method.")
+            print("Trying to remesh with alternative method.")
```

Comment:
Sorry, I only meant to change the last part of the message. I think this is fine:
``` python
print("ERROR: Mesh generation failed. Trying to remesh with alternative method.")
```
Simply replace `Try` with `Trying`.